scrapyd_k8s/config.py

```diff
@@ -21,13 +21,27 @@ def launcher_cls(self):
         pkg, cls = repo.rsplit('.', 1)
         return getattr(import_module(pkg), cls)
 
+    def joblogs(self):
+        if self._config.has_section('joblogs'):
+            return self._config['joblogs']
+        else:
+            return {}
```

Comment:
This can be written shorter as `self._config.get('joblogs', {}), right?
Or it could also make sense to return `None` if no joblogs section is present, I think that would be clearer perhaps?

Reply:
I agree that return None makes more sense in this case